src/expr/src/relation/mod.rs

```diff
@@ -16,6 +16,7 @@ use std::fmt;
 use itertools::Itertools;
 use serde::{Deserialize, Serialize};
 
+use lowertest::MzEnumReflect;
```

Comment:
Random thought... Should we not generate the reflection information when compiling the test config? My worry being that unused code slowing code the rest of the configs. It might be neglectable though. Out of curiosity, have you run a compile time test before and after this patch? Anyway, I'm sure it's neglectable, but asking just in case...

Reply:
I have not tried running a compile time test, but I will. 

I originally tried using `#[cfg(test)]` to that way the macro is only generated when compiling a test, but it turns out that if you use `#[cfg(test)]`, you only generate the macro for `expr/tests` if the macro is on an enum/struct in `expr`, and the macro becomes invisible to `transform/tests`.

That said, I just had an idea - a way we could generate reflection information only at test time is to search the .rs files for the text corresponding to the struct/enum, turn the text into a `proc_macro2::TokenStream`, and then feed that `proc_macro2::TokenStream` into the existing reflection-generation method.

Reply:
Personal take is that it's not worth worrying about. Compile time in Rust is sooo dominated by LLVM codegen that it is extremely unlikely that a proc macro could meaningfully impact compile times. 

FWIW, `walkabout` takes the approach you describe, and parses `.rs` files from source at compile time. It does this out of necessity, because it needs to consider multiple structs/enums at one time and proc macros don't allow for that. I wouldn't recommend this path if you can help it—reinventing the logic for parsing `mod` declarations sucks, and walkabout's implementation is incomplete—but that logic exists if you need it. I think `walkabout::parse_mod` is actually _exactly_ what you'd need.

Reply:
Is `walkabout::parse_mod` public? If not, would you be okay with making it public?

Reply:
Yeah, that seems fine for now. One day we may want a `proc-macro-util` crate or somesuch.